db/re/item_db_equip.yml

```diff
@@ -148071,6 +148270,15 @@ Body:
             }
          }
       }
+  - Id: 480158
+    AegisName: C_Lunatic_Bag
+    Name: Costume Lunatic Bag
+    Type: Armor
+    Locations:
+      Costume_Garment: true
+    ArmorLevel: 1
+    EquipLevelMin: 1
```

Comment:
```
    ArmorLevel: 1
    EquipLevelMin: 1
```
Are redundant, because ArmorLevel defaults to 1 for Armors, and minimum level of any char is 1.
Throughout all your changes/additions of costumes above.